### PR TITLE
chore(travis): re-enabled auto-release

### PR DESCRIPTION
**Pull request by nickbalestra**
This PR to re-enable semantic-release, as discussed on #94.
Needed tokens are encrypted directly on the `travis.yml`

As the GH token is also required for `semantic-release`, and as I've not yet write access to this repo, I guess that auto-release could still potentially fail ..

### CODE REVIEW

**Comment by nickbalestra** [comment]
@Garbee @kentcdodds - I've added the needed tokens directly on the travis.yml but I guess I need to have publish rights both on npm && on this repo on GH in order for auto-release to work ...?

If that's the case once i get both those rights, this PR should work and address #94

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,22 @@ sudo: false
 language: node_js
 cache:
   directories:
-    - node_modules
+  - node_modules
```
Comment (stevemao) [inline .travis.yml]: I think this is wrong indentation?

Comment (nickbalestra) [inline .travis.yml]: mmm, travis CLI did that when I added the encrypted tokens through it `travis encrypt --add` so I thought it should be ok...

**Comment by Garbee** [comment]
Just invited you to get write access to the repo to make sure this has the best chance at working.

File: .travis.yml (hunk 1 above)
Comment (Garbee) [inline .travis.yml]: It will work either way. Indentation is preferred though for improved readability.

**Comment by nickbalestra** [comment]
Thanks @Garbee I've also updated the PR with the suggested indentation.
Over to you for merging and 🤞

**Comment by Garbee** [comment]
LGTM. Let's merge and see.

**Comment by stevemao** [comment]
@nickbalestra I invited you to the org. welcome onboard :)

**Comment by nickbalestra** [comment]
Thank you @stevemao !!

**Pull request merged**